src/main/webapp/browsers/tswa.html

```diff
@@ -1,12 +1,11 @@
 <div>
   <p>
-	Enter the URL to the TSWA server. Examples:<tt>http://tswasrv</tt> or <tt>http://TFSMachine:8080/</tt>.
-	The plugin will strip the last path (if any) of the URL when building URLs for change set pages and other pages. 
+	Enter the URL to your project on the TFS server, used for browser-based access.

```

Comment:
We should be more specific as to why someone would need to provide a value here.  In other words, when is the default/derived value not acceptable?